resources/js/common/components/PlayableListSortButton/PlayableListSortButton.tsx

```diff
@@ -68,6 +69,9 @@ export const PlayableListSortButton: FC<PlayableListSortButtonProps> = ({
 
     '-wonBy': { label: t('Won by (least)'), icon: LuUsers },
     wonBy: { label: t('Won by (most)'), icon: LuUsers },
+
+    '-rank': { label: t('Rank (lowest)'), icon: LuHash },
+    rank: { label: t('Rank (highest)'), icon: LuHash },
```

Comment:
UX: I'm wondering if we should reconsider these labels. When we sort by "Rank (highest)", we're technically sorting by the lowest number 😅 

<img width="857" height="301" alt="Screenshot 2025-10-30 at 6 12 06 PM" src="https://github.com/user-attachments/assets/4f7e43b7-727d-427f-ad45-b6aa9301262e" />

Possible alternatives:
* Best / Worst
* Top / Bottom